src/IssueViz/Commands/IssueVizPackage.vsct

```diff
@@ -22,6 +22,28 @@
         </Strings>
       </Button>
 
+      <Button guid="guidIssueVizPackageCmdSet" id="GotoNextLocationCommandId" priority="0x0100" type="Button">
+        <Parent guid="guidIssueVizPackageCmdSet" id="SonarLintMenuGroup" />
+        <CommandFlag>CommandWellOnly</CommandFlag>
+        <Strings>
+          <ButtonText>Go to next issue location</ButtonText>
+          <CanonicalName>SonarLintGoToNextLocation</CanonicalName>
+          <LocCanonicalName>SonarLintGoToNextLocation</LocCanonicalName>
+          <CommandName>SonarLint Go to next issue location</CommandName>
+        </Strings>
+      </Button>
+
+      <Button guid="guidIssueVizPackageCmdSet" id="GotoPreviousLocationCommandId" priority="0x0100" type="Button">
```

Comment:
Specifying two commands in the same group with the same priority doesn't make sense, but it probably doesn't matter since the commands are never displayed in the UI.

Reply:
Yep, I read that if two commands have the same priority, then they are just rendered in different order (which wouldn't matter, as they're not shown anyway).